src/sql/src/plan/statement.rs

```diff
@@ -162,6 +162,15 @@ pub fn describe_statement(
             vec![],
         ),
 
+        Statement::ShowCreateIndex { .. } => (
+            Some(
+                RelationDesc::empty()
+                    .with_nonnull_column("Index", ScalarType::String)
+                    .with_nonnull_column("Create Index", ScalarType::String),
```

Comment:
These kinds of column names weird me out a little bit, but I guess it's fine/normal since we don't have a mechanism for people to read from a `show` statement anyway?

Reply:
Yeah, they're weird but just following the existing patterns.